### PR TITLE
Log file

### PR DESCRIPTION
**Pull request by bitflipp**

### CODE REVIEW

File: logging.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	now() time.Time
+}
+
+type realTimeClock struct{}
+
+func (c *realTimeClock) now() time.Time {
+	return time.Now()
+}
+
+type rfcPrefixWriter struct {
```
Comment (Top-Ranger) [inline logging.go:18]: Would it be possible to use an interface here? Maybe with some kind of factory? That would make it possible to add different formats later

(for example, I'm thinking of a logger which uses JSON format for easier parsing).

File: logging_test.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+type mockClock struct {
+	time time.Time
+}
+
+func (c *mockClock) now() time.Time {
+	return c.time
+}
+
+func TestRFCPrefixWriter(t *testing.T) {
```
Comment (Top-Ranger) [inline logging_test.go:17]: Idea: Use more than one line for test.
That would make the test more robust - just in case.

**Review by Top-Ranger — APPROVED** [review]

File: logging.go (hunk 1 above)
Comment (bitflipp) [reply logging.go:18]: When we choose to implement structured logging, we should consider using dedicated libraries like [Zerolog](https://github.com/rs/zerolog).

Comment (Top-Ranger) [reply logging.go:18]: Would it make sense to do it correctly now?
(And plan a major release for it?)

**Pull request merged**